packages/webapp/src/utils/credits-purchase-constants.js

```diff
@@ -1,12 +1,20 @@
 const routes = {
-  ESTIMATOR_CREDITS_INDIVIDUAL_ORG: '/credits-purchase/applying-individual-organisation'
+  // CREDITS_PURCHASE_APPLYING_INDIVIDIAL_ORGANISATION: '/credits-purchase/applying-individual-organisation',
```

Comment:
I'm happy for this line to be commented as a reminder until https://github.com/DEFRA/biodiversity-net-gain-service/pull/615 is merged.